examples/rockpaperscissors/agent.py

```diff
@@ -1,17 +1,27 @@
-from comprl.client.agent import COMPAgent
+from comprl.client import Agent
+import random
 
 
-class RPSAgent(COMPAgent):
-    """Dummy Agent for testing. Returns random action"""
+RPSAgent = Agent()
```

Comment:
RPSAgent is an object at this point it should be something like agent or so (see [here](https://google.github.io/styleguide/pyguide.html#s3.16-naming ))